source-mysql-batch/driver.go

```diff
@@ -575,28 +309,105 @@ func (c *capture) Run(ctx context.Context) error {
 	return nil
 }
 
+func (c *capture) emitSourcedSchemas(ctx context.Context) error {
+	if !c.Config.Advanced.parsedFeatureFlags["emit_sourced_schemas"] {
+		return nil
+	}
+
+	c.shared.Lock()
+	defer c.shared.Unlock()
+
+	// Skip re-running discovery and emitting schemas if we just did it.
+	if time.Since(c.shared.lastDiscoveryTime) < minDiscoveryInterval {
```

Comment:
Is this `lastDiscoveryTime` ever getting set? I can't seem to find where it is getting set to "now", but I would expect it to get set a little further down in this function.

Reply:
:man_facepalming: Good catch, it's never getting set